src/Controls/src/Core/Platform/Android/SwipeGestureHandler.cs

```diff
@@ -36,34 +38,65 @@ public bool OnSwipe(float x, float y)
 			if (view == null)
 				return false;
 
+			var transformedCoords = TransformSwipeCoordinatesWithRotation(x, y, view.Rotation);
+
+			_totalX += PixelTranslation(transformedCoords.x);
+			_totalY += PixelTranslation(transformedCoords.y);
+
 			var result = false;
 			foreach (SwipeGestureRecognizer swipeGesture in
-					 view.GestureRecognizers.GetGesturesFor<SwipeGestureRecognizer>())
+					view.GestureRecognizers.GetGesturesFor<SwipeGestureRecognizer>())
 			{
-				((ISwipeGestureController)swipeGesture).SendSwipe(view, PixelTranslation(x), PixelTranslation(y));
+				((ISwipeGestureController)swipeGesture).SendSwipe(view, _totalX, _totalY);
 				result = true;
 			}
 
 			return result;
 		}
 
+		(float x, float y) TransformSwipeCoordinatesWithRotation(float x, float y, double rotation)
+		{
+			var correctedX = x; 
+			var correctedY = y;  
+
+			if (Math.Abs(rotation) < 0.01)
```

Comment:
Can you add a comment here? Or, create a constant with the 0.01 value and give a descriptive name to it.

Reply:
I have introduced a named constant for 0.01 with a descriptive name and added a comment for clarity.